design-library/source/_patterns/02-molecules/08-list-post/list-post.twig

```diff
@@ -0,0 +1,10 @@
+<div class="list-post">
```

Comment:
I'm surprised there are two wrapping div tags

Reply:
I will fix this